docs/guides/code_push_release_app_store.md

```diff
@@ -0,0 +1,161 @@
+---
+sidebar_position: 3
+title: 🍎 Release a Code Push App (iOS)
+description: Release a Code Push App (iOS)
+---
+
+# Releasing a code push app to the Apple App Store
+
+This guide walks through releasing a code push app to the Apple App Store and applying a patch to that release.
+
+The app we will be releasing in this guide is [`Shorebird Clock`](TODO), our demo code push app. ([source](https://github.com/shorebirdtech/time_shift/))
```

Comment:
I think this will cause trouble for you since it already has a shorebird app_id, a (unique) apple bundle_id, and a development team set.  You might want an example that has none of those.

Reply:
I'll update the prereqs to call these out, but I tried to refer to Apple's documentation where possible as a lot of this is not specific to Shorebird. To address the individual points:

1. shorebird app_id: I didn't want to duplicate our "getting started" guide, so I pointed users there in the prereqs
2. (unique) apple bundle_id: Pointed users to the official apple docs at https://developer.apple.com/help/app-store-connect/create-an-app-record/add-a-new-app
3. a development team set: Will update